pkg/operator/controller/garden/garden/reconciler_reconcile.go

```diff
@@ -1068,6 +1069,25 @@ func (r *Reconciler) deployGardenPrometheus(ctx context.Context, log logr.Logger
 		}
 	}
 
+	seeds := []string{}
+	for _, seed := range seedList.Items {
+		var seedName = seed.Name
+		seeds = append(seeds, seedName)
+	}
+	joinedSeeds := strings.Join(seeds, "|")
+
+	additionalAlertRelabelConfigs := []*monitoringv1.RelabelConfig{
+		{
+			SourceLabels: []monitoringv1.LabelName{"project", "name"},
+			Regex:        ";" + joinedSeeds,
```

Comment:
opinionated: I'd probably inline `joinedSeeds` to make it slightly easier to understand what the constructed regex contains
```suggestion

	additionalAlertRelabelConfigs := []*monitoringv1.RelabelConfig{
		{
			SourceLabels: []monitoringv1.LabelName{"project", "name"},
			Regex:        ";" + strings.Join(seeds, "|"),
```